test/infrastructure/docker/Makefile

```diff
@@ -113,6 +113,9 @@ generate-go: $(CONTROLLER_GEN) $(CONVERSION_GEN) ## Runs Go related generate tar
 		paths=./$(EXP_DIR)/api/...
 	$(CONVERSION_GEN) \
 		--input-dirs=./api/v1alpha3 \
+		--input-dirs=./$(EXP_DIR)/api/v1alpha3 \
+		--build-tag=ignore_autogenerated_infra_v1alpha3 \
```

Comment:
```suggestion
		--build-tag=ignore_autogenerated_capd_v1alpha3 \
```

Reply:
Maybe?

Reply:
I kept it as `infra` since the imports in the Go files were prefixed with the same.

Reply:
Changed it to `capd`